### ISSUE TITLE
Expo SDK 47 Support

### ISSUE THREAD

**Issue by Nantris** [jakobo/expo-community-flipper#40]
Thanks for all the great work on this project! I wondered if there will be substantial modifications necessary for Expo SDK 47?

**Comment by Nantris** [comment]
My observations so far:

1. Android app - Flipper connects with the app, but Hermes Debugger does not
2. iOS EAS builds fail:

```
- Config syncing
✖ Config sync failed
[stderr] Error: [ios.dangerous]: withIosDangerousBaseMod: Cannot add flipper arguments to the project's ios/Podfile. Please report this with a copy of your project Podfile. You can generate this with the `expo prebuild` command.
[stderr] Error: [ios.dangerous]: withIosDangerousBaseMod: Cannot add flipper arguments to the project's ios/Podfile. Please report this with a copy of your project Podfile. You can generate this with the `expo prebuild` command.
[stderr]     at /Users/expo/workingdir/build/node_modules/expo-community-flipper/build/withFlipper.ios.js:120:23
[stderr]     at action (/Users/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/withMod.js:235:29)
[stderr]     at interceptingMod (/Users/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/withMod.js:126:27)
[stderr]     at action (/Users/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/withMod.js:240:14)
[stderr]     at async interceptingMod (/Users/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/withMod.js:126:21)
[stderr]     at async interceptingMod (/Users/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/withMod.js:126:21)
[stderr]     at async interceptingMod (/Users/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/withMod.js:126:21)
[stderr]     at async interceptingMod (/Users/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/withMod.js:126:21)
[stderr]     at async action (/Users/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/createBaseMod.js:71:21)
[stderr]     at async interceptingMod (/Users/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/withMod.js:126:21)
[stderr]     at async evalModsAsync (/Users/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/mod-compiler.js:256:25)
[stderr]     at async Object.compileModsAsync (/Users/expo/workingdir/build/node_modules/@expo/config-plugins/build/plugins/mod-compiler.js:160:10)
[stderr]     at async configureProjectAsync (/Users/expo/workingdir/build/node_modules/@expo/cli/build/src/prebuild/configureProjectAsync.js:54:15)
[stderr]     at async prebuildAsync (/Users/expo/workingdir/build/node_modules/@expo/cli/build/src/prebuild/prebuildAsync.js:83:9)
[stderr] error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by Nantris** [comment]
I just noticed PR #37, but I guess it's not published yet. I wonder if it will fix the issue with EAS build for iOS?

**Comment by thecodedrift** [comment]
It will. If you can try out the changes locally that'd be a huge help. We're just finalizing a few things in the PR before we make it available for those trying the 47 beta

**Comment by thecodedrift** [comment]
Fixed in #38 and released as v47.0.0

**Comment by Nantris** [comment]
Whoops sorry @jakobo! I didn't see your previous message. Thanks for your great work!
